index.js

```diff
@@ -17,13 +17,20 @@ if (testSupport()) {
   storageEngine = localStorage
 }
 
+let pageShowHandler
+const backFromBfCache = sync => () => sync()
```

Comment:
Why do we need this function?

Reply:
We don't. I realized that we can simply assign pageShowHandler to `() => sync()`
Fixed that